test/cli.js

```diff
@@ -69,3 +69,12 @@ test('quiet option', async t => {
 	const err = await t.throws(execa('../cli.js', ['--no-local', '--quiet', '--reporter=compact', filepath]));
 	t.is(err.stdout.indexOf('warning'), -1);
 });
+
+test('init option', async t => {
+	const filepath = await tempWrite('{}', 'package.json');
+	await execa(path.join(__dirname, '../cli.js'), ['--init'], {
+		cwd: filepath.replace('package.json', '')
+	});
+	const packagejson = await fs.readFileSync(filepath, 'utf8');
+	t.regex(JSON.stringify(JSON.parse(packagejson)), /{"scripts":{"test":"xo"},"devDependencies":{"xo":"\^[0-9]+\.[0-9]+\.[0-9]+"}}/);
```

Comment:
Wouldn't that mean only scripts could be checked since we can't hardcode the version number or should I grab the version number from `../package.json` and test against that?